package.json

```diff
@@ -162,5 +163,20 @@
         "npm run test -- --bail --findRelatedTests"
       ]
     }
-  }
+  },
+  "//": "browserslist set to 'Async functions' compatibility",
+  "browserslist": [
```

Comment:
How about using configuration like we had before (`> 0.5%, last 2 versions, Firefox ESR, ie 11, not dead`) so we don't have to update this list everytime browser versions update? I mean not necessarily the same one, but along the lines of `> 0.5%, last 2 versions` etc instead of hardcoded versions.

Reply:
> so we don't have to update this list every time browser versions update

Why would you need to? We should revisit this list when:
1. There are lower browser versions we would like to support. Won't happen cause we've looked at the browser stats.
2. As a performance optimization to lower `app.js` file size. That can be done periodically.

> instead of hardcoded version

The advantage of hardcoded versions is that they allow to easily and reliably determine if a user's browser is supported or not. But if we define sth like "last 2 versions" you would have no clear cutoff point (cause under last 2 versions doesn't mean you can't use Redash).